android/KMEA/app/src/main/assets/languages/chirality.js

```diff
@@ -0,0 +1,79 @@
+﻿KeymanWeb.KR(new Keyboard_chirality());
```

Comment:
Why is this file in the KMEA assets and not in a test folder?

Reply:
chirality.js is temporarily here while we troubleshoot the stability of the chiral test. I will remove before merging